docs/cloud-provider-specifics.md

```diff
@@ -31,6 +31,14 @@ aws s3api create-bucket \
     --region <YOUR_REGION> \
     --create-bucket-configuration LocationConstraint=<YOUR_REGION>
 ```
+Note: us-east-1 does not require a `location-constraint`.  If your region is `us-east-1`, omit the bucket configuration:
```

Comment:
This phrasing reads as if `location-constraint` is optional, but I believe the issue is that using it causes creation to fail.

Perhaps something like "us-east-1 does not support a `location-constraint`".  Any thoughts @Bradamant3?

Reply:
Location-constraint is optional in the sense that the command with the flag removed successfully creates the bucket; however, not having the constraint may break a downstream dependency.

```
$ aws s3api create-bucket --bucket bar --region us-east-1
{
    "Location": "/bar"
}
```

Reply:
Ah, @nrb, I now savvy your semantic distinction.  I'll edit my pr.